opentelemetry-exporter-google-cloud/tests/test_cloud_monitoring.py

```diff
@@ -414,6 +416,59 @@ def test_export_value_observer(self):
             [mock.call(self.project_name, [series])]
         )
 
+    def test_export_histogram(self):
+        client = mock.Mock()
+
+        with mock.patch(
+            "opentelemetry.exporter.cloud_monitoring.time_ns", lambda: int(1e9)
+        ):
+            exporter = CloudMonitoringMetricsExporter(
+                project_id=self.project_id, client=client
+            )
+
+        exporter.project_name = self.project_name
+
+        client.create_metric_descriptor.return_value = MetricDescriptor(
+            **{
+                "name": None,
+                "type": "custom.googleapis.com/OpenTelemetry/name",
+                "display_name": "name",
+                "description": "description",
+                "labels": [],
+                "metric_kind": "CUMULATIVE",
+                "value_type": "DISTRIBUTION",
+            }
+        )
+
+        aggregator = HistogramAggregator(config={"bounds": [2, 4, 6]})
+        aggregator.checkpoint = OrderedDict([(2, 1), (4, 2), (6, 4), (">", 3)])
```

Comment:
Could you fill me in on the `>` -- why is there no `<`?

Reply:
the current histogram aggregator defines buckets as everything < than bound[i] and >= than bound[i-1] - so you have a > for everything over the last bucket but your first bucket already covers everything below. This may change after the SDK spec is defined